compilers/concrete-compiler/compiler/lib/Bindings/Python/concrete/compiler/client_support.py

```diff
@@ -82,8 +86,10 @@ def key_set(
         Returns:
             KeySet: generated or loaded keyset
         """
-        assert 0 <= seed_msb < 2**64
-        assert 0 <= seed_lsb < 2**64
+        assert 0 <= secret_seed_msb < 2**64
```

Comment:
raise ValueError("blabla")

Reply:
assertion is not the right exception here

Reply:
yes just copy/paste previous logic but you are right